Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -75,8 +103,46 @@ public override void Serialize(IIonWriter writer, object item)
         {
             options.TypeAnnotator.Apply(options, writer, targetType);
             writer.StepIn(IonType.Struct);
+
+            var serializedIonFields = new HashSet<string>();
+            
+            // Serialize the values returned from IonPropertyGetter annotated getter methods.
+            foreach (var method in targetType.GetMethods())
+            {
+                var getMethod = (IonPropertyGetter)method.GetCustomAttribute(typeof(IonPropertyGetter));
+                
+                // A getter method should have zero parameters.
+                if (getMethod?.IonPropertyName == null || method.GetParameters().Length != 0)
+                {
+                    continue;
+                }
+
+                var getValue = method.Invoke(item, Array.Empty<object>());
+                if (options.IgnoreNulls && getValue == null)
```

Comment:
`IgnoreNulls` does not apply to methods.

Reply:
Removed `IgnoreNulls` logic.